tests/spec/aggregate_series/test_count_distinct.py

```diff
@@ -0,0 +1,66 @@
+from ..tables import e
+
+
+title = "Count distinct aggregation"
+
+table_data = {
+    e: """
+          |  i1 |  f1 | s1 |     d1
+        --+-----+-----+----+------------
+        1 | 101 | 1.1 |  a | 2020-01-01
+        1 | 102 | 1.2 |  b | 2020-01-02
+        1 | 103 | 1.5 |  c | 2020-01-03
+        2 | 201 | 1.1 |  a | 2020-02-01
+        2 | 201 | 1.1 |  a | 2020-02-01
+        2 | 203 | 1.5 |  b | 2020-02-02
+        3 |     |     |    |
```

Comment:
I think the test cases need to include a patient who has a mixture of NULL and non-NULL values. We want to ensure that, e.g., `101`, `101`, `NULL`, `NULL` is counted as 1 distinct value not 3 (or even 4, some databases regard every NULL value as unique).

Reply:
good idea!